src/main/java/it/gov/pagopa/pu/workflow/wf/assessments/wfassessments/CreateAssessmentsWF.java

```diff
@@ -0,0 +1,15 @@
+package it.gov.pagopa.pu.workflow.wf.assessments.wfassessments;
+
+import io.temporal.workflow.WorkflowInterface;
+import io.temporal.workflow.WorkflowMethod;
+
+/**
+ * Workflow interface for the Assessment creation
```

Comment:
see the other WF javadoc, let's add link to Confluence page
https://pagopa.atlassian.net/wiki/spaces/SPAC/pages/1621491884/Creazione+Accertamenti